Adds a missing header to Appearance.h that prevents it from being directly
#include'd in downstream code

---

diff --git a/third_party/libosim/opensim-core/OpenSim/Simulation/Model/Appearance.h b/third_party/libosim/opensim-core/OpenSim/Simulation/Model/Appearance.h
index 5c8206f9b..cae919174 100644
--- a/third_party/libosim/opensim-core/OpenSim/Simulation/Model/Appearance.h
+++ b/third_party/libosim/opensim-core/OpenSim/Simulation/Model/Appearance.h
@@ -28,6 +28,7 @@
 #include <OpenSim/Simulation/osimSimulationDLL.h>
 #include <OpenSim/Common/Object.h>
 #include <OpenSim/Common/XMLDocument.h>
+#include <SimTKcommon/internal/DecorativeGeometry.h>
 
 namespace OpenSim {
 
